customisation/glossaries.md

```diff
@@ -5,3 +5,27 @@ layout: coming_soon
 title: Glossaries
 description: Terminology lists for training and customising machine translation
 ---
+#Glossaries
+
+Glossaries, i.e. lists of terms in one language aligned to their translation in one or more languages, can be added to machine translation systems.
+When a glossary term is matched in the source sentence, its translation is enforced in the output.
+
+## How glossaries are leveraged by MT systems
+Glossaries are leveraged at translation time, and no training is required.
+Adding a glossary to a commercial MT provider can increase the terminology accuracy of the output.
+However, the insertion of a term in the output happens regardless of the context.
+The terms included in the glossary should therefore be unambiguous, meaning that their translation should not be heavily dependent on the context.
+For example, "respiratory syndrome" is a good candidate to appear in a glossary, whereas the translation of the word "spread" may differ when it is used as a verb or a noun, or when the term occurs in the medical or in the financial domain.
```

Comment:
What do you think of creating a section called `Examples` as we have [here](https://machinetranslate.org/language-model)?